packages/forms-web-app/src/views/interested-party-guide/get-involved-preliminary-meetings.njk

```diff
@@ -5,13 +5,13 @@
 {% endblock %}
```

Comment:
It is the same with the font here as well. If you compare [https://applications-prototype.herokuapp.com/mvp/interested-party-guide/what-you-can-do-after-decision](https://applications-prototype.herokuapp.com/mvp/interested-party-guide/what-you-can-do-after-decision) and [http://localhost:9004/interested-party-guide/what-happens-after-decision](http://localhost:9004/interested-party-guide/what-happens-after-decision). I inspected the pages in the browser, but could not see difference. It is probably the same for the pages I worked on, but I notice it only now. Do we ignore this as display issue ?

Reply:
Would you please paste confirmation link ?

Reply:
Which confirmation link? https://design-system.service.gov.uk/styles/typography/ here is font style.